### PR TITLE
Feature/274 release guide

### LINKED ISSUE #277: Don't fail on git tags with unsupported version identifier
For https://github.com/exasol/cloud-storage-extension/tags?after=v0.6.2-rc1
`release droid --local . --goal validate` currently fails with message 
> Failed reading version from git tag 'refs/tags/v0.6.0-7-g451d30c-SNAPSHOT'.

### PR DESCRIPTION
**Pull request by ckunki**
fixes #274 
fixes #277

### CODE REVIEW

File: src/main/resources/release-guide-template.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+.error {
+		color: #f00;
+		font-style: italic;
+}
+
+code {
+		background-color: #eeeeee;
+		padding: 1px;
+		xcolor: #055;
+		xfont-style: normal;
```
Comment (kaklakariada) [inline src/main/resources/release-guide-template.html:73]:
```suggestion
	font-size: 80%;
	color: #666;
	vertical-align: bottom;
	font-style: italic;
}

.error {
	color: #f00;
	font-style: italic;
}

code {
	background-color: #eeeeee;
	padding: 1px;
	xcolor: #055;
	xfont-style: normal;
```

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: doc/user_guide/user_guide.md [hunk 2]
```diff
@@ -122,8 +122,11 @@ jira_username=<your jira user name>
 jira_password=<jira password>
 ```
 
-The last to lines are only relevant if releasing to platform Jira.<br />
-If Release Droid cannot find this file during an execution, it asks the user to input the credentials directly through terminal.<br />
+The lines with prefix "jira" are only relevant if releasing to platform Jira.
```
[the remark this answers is not in the document]
Comment (ckunki) [reply doc/user_guide/user_guide.md:125]: Yes, but the functionality is still implemented. I agreed to not removing it, yet.

File: src/main/java/com/exasol/releasedroid/adapter/repository/LocalRepositoryGate.java [hunk 3]
```diff
@@ -36,6 +37,20 @@ static Optional<Version> latestTagFromRefs(final List<Ref> refs) {
...
+    /**
+     * Create a {@link LocalRepositoryGate} base on a working copy in a local folder and retrieve the full name of the
+     * (remote) repository from its Git metadata.
+     *
+     * @param folder folder containing the files of the local repository
+     * @return new instance of {@link LocalRepositoryGate}
+     * @throws IOException if accessing the Git metadata of the local repository fails.
+     */
+    public static LocalRepositoryGate from(final Path folder) throws IOException {
+        final String name = new RemoteName(Git.open(folder.toFile())).retrieve()
```
[the remark this answers is not in the document]
Comment (ckunki) [reply src/main/java/com/exasol/releasedroid/adapter/repository/LocalRepositoryGate.java:49]: Ups - yes, thanks!

**Review by ckunki — COMMENTED** [review]
reviewed findings

**Review by kaklakariada — APPROVED** [review]

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**